hack/lib/start.sh

```diff
@@ -292,8 +292,10 @@ function os::start::master() {
 	echo "[INFO] OpenShift server start at: "
 	date
 
-	wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz" "apiserver: " 0.25 160
-	wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz/ready" "apiserver(ready): " 0.25 160
+	os::test::junit::declare_suite_start "os::start::master"
```

Comment:
We don't use colons as separators in the jUnit suites - we can just have `setup/start-master`